tensorboard/components/tensor_widget/README.md

```diff
@@ -19,3 +19,24 @@ Features
 
 *This component is in early stage of active development.*
 *Many of its features are still incomplete.*
+
+## Developer workflow
+
+### Running the development server
+
+1. Make sure you have installed bazel and optionally also
+   [ibazel](https://www.npmjs.com/package/@bazel/ibazel)
```

Comment:
honestly, I think this is a better pointer: https://github.com/bazelbuild/bazel-watcher#installation
(not everyone use node/npm and global installation; `npm i -g`)

Reply:
Good suggestion. Done.